pkg/es/writer/esBulkHandler_test.go

```diff
@@ -77,11 +77,11 @@ func Test_IngestMultipleTypesIntoOneColumn(t *testing.T) {
 		[]byte(`{"age": "171"}`),
 		[]byte(`{"age": 103}`),
 		[]byte(`{"age": 5.123}`),
-		[]byte(`{"age": true}`),
+		// []byte(`{"age": true}`),
```

Comment:
Is this related to the same comment that Andrew has made? If yes, can we not fix the issue? or can you add a TODO that this issue needs to be fixed?

Reply:
I will add an issue in the backlog